src/common/test/componentMocking.tsx

```diff
@@ -60,11 +60,12 @@ export type TestTools = RenderResult & {
     selector: ElementSelector
```

Comment:
File comment: This file is quite large. It seems to contain test related helper functions and types? I just wonder if it's doing too much, as the name componentMocking feels like it does not match for all what the file exposes. For example functions getElementAttribute and waitForElementAttributeValue. The feel like helper functions for testing, not specifically about mocking components? Not sure about how it could be improved though, just an observation 😅

Reply:
Yep. 👍   

Could split types, helpers and rendering?

Reply:
Done.